core/src/main/java/org/kiwix/kiwixmobile/core/dao/HistoryDao.kt

```diff
@@ -42,8 +43,16 @@ class HistoryDao @Inject constructor(val box: Box<HistoryEntity>) : PageDao {
     box.store.callInTx {
       box
         .query {
-          equal(HistoryEntity_.historyUrl, historyItem.historyUrl).and()
-            .equal(HistoryEntity_.dateString, historyItem.dateString)
+          equal(
+            HistoryEntity_.historyUrl,
+            historyItem.historyUrl,
+            QueryBuilder.StringOrder.CASE_INSENSITIVE
+          ).and()
+            .equal(
+              HistoryEntity_.dateString,
+              historyItem.dateString,
+              QueryBuilder.StringOrder.CASE_INSENSITIVE
+            )
```

Comment:
Hi @gouri-panda ,
After upgrading ```objectbox``` from ```2.9.1``` to ```3.5.0``` this is necessary to pass ```StringOrder``` otherwise it will give compile time error.